src/equations/2d/compressible_euler_multicomponent.jl

```diff
@@ -0,0 +1,980 @@
+
+@doc raw"""
+    CompressibleEulerMulticomponentEquations2D
+
+The compressible Euler Two!-Component equations for an ideal gas in two space dimensions.
+"""
+struct CompressibleEulerMulticomponentEquations2D <: AbstractCompressibleEulerMulticomponentEquations{2, 5}
+  # CHRIS: Hier muss gamma ein Array der Größe 2 sein
+  gamma::Float64
+end
+
+function CompressibleEulerMulticomponentEquations2D()
+  # CHRIS: Gamma ist nun ein Array, muss mehrere Werte aus Parameter File lesen
+  gamma = parameter("gamma", 1.4)
+
+  # CHRIS: Hier vllt extra Parameter 'K_numb_of_comp'
```

Comment:
I think number of components should be a parameter to `AbstractCompressibleEulerMulticomponentEquations` such that it is a compile-time constant and can be used as such.